stellargraph/mapper/full_batch_generators.py

```diff
@@ -238,3 +238,209 @@ def flow(self, node_ids, targets=None):
             return FullBatchNodeSequence(
                 self.features, self.Aadj, targets, node_indices
             )
+
+
+class RelationalFullBatchNodeSequence(Sequence):
```

Comment:
Move this class to the `mapper/sequences.py` file.

Reply:
done!